updates to tealeg/xlsx result in gocsv build errors

`go get github.com/DataFoxCo/gocsv` now results in:
```
src/github.com/DataFoxCo/gocsv/cmd/xlsx.go:105:27: sheet.Rows undefined (type *xlsx.Sheet has no field or method Rows)
```

--- a/go.mod
+++ b/go.mod
@@ -8,4 +8,5 @@ replace (
 require (
 	github.com/aotimme/gocsv/cmd v0.0.0
 	github.com/aotimme/gocsv/csv v0.0.0
+	github.com/tealeg/xlsx v1.0.5
 )

